### DISCUSSION TITLE
Welcome to vue-data-ui Discussions!

### DISCUSSION THREAD

**Discussion by graphieros** [graphieros/vue-data-ui#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KayElliot** [comment]
Introducing myself!

I work at a small SaaS company that is using CodeIgniter 3.x on a LAMP stack and about a year ago (or two?) we made the commitment to start integrating VueJS in our software. As our CI/CD pipeline did not allow us to integrate webpack, and my inexperience with VueJS at the time, we opted to use the Options API. Besides my regular job, I've also just started building my first solo SaaS in the evenings. This is done using a combination with Laravel + VueJS with InertiaJS which is quite the opposite of my professional stack.

I stumbled upon this library whilst searching for an alternative to Vuesalize, which I chose two days ago but sadly had to conclude that its styling capabilities were not suited for what I have in mind. My first impression of vue-data-ui is that designwise it's very impressive. Nevertheless, I am yet to get it up and running in VueJS Options API while using a CDN.

My plans for this library are to help with the creation of an installation guide for people who use VueJS Options API through a CDN whilst simultaneously reporting bugs to you, the creator. This way, I hope to aid in the process of making this library available to a larger audience. Additionally, I will integrate the library into a few of our back-end administration screens, and collect feedback from colleagues who work with the visual information this library provides. I will communicate this feedback to you, the creator.

As for now, I still need to get this library up and running within our company's application, after which I will report back wih the installation guide written in Markdown. I am new to the actual community of GitHub but I've lurked around since I started developing, so I will have to figure out how to properly provide you with the files related to the installation guide.

Kind regards,

Kay Elliot

**Reply by graphieros** [reply]
HI Kay,

Thanks for the introduction and the good words :)

I hope we can find a way to make this work through a CDN, I have to admit I overlooked this usage, as I was focused on the traditional package install.

Cheers,

Alec
